### PR TITLE
iio: Remove warnings caused when building with make

### PR DESCRIPTION
**Pull request by cristiantpop**
Signed-off-by: Cristian Pop <cristian.pop@analog.com>

### CODE REVIEW

**Comment by cristiantpop** [comment]
V2:

- Add second commit: warning: "EOPNOTSUPP" redefined

File: projects/ad9361/src/common.h [hunk 1]
```diff
@@ -42,19 +42,13 @@
 /******************************************************************************/
 /***************************** Include Files **********************************/
 /******************************************************************************/
+
 #include <stdint.h>
+#include <errno.h>
 
 /******************************************************************************/
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
```
Comment (amiclaus) [inline projects/ad9361/src/common.h]: nitpick: you can also remove the header `Macros and Constants Definition`

**Comment by cristiantpop** [comment]
V3:

- remove the header Macros and Constants Definition

File: projects/ad9361/src/common.h (hunk 1 above)
Comment (cristiantpop) [reply projects/ad9361/src/common.h]: Done

**Pull request merged**